docs/UserGuide.md

```diff
@@ -27,7 +27,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `add n/Jun Jie p/98765432 e/jj@example.com a/Clementi Ave 3, block 442, #06-01` : Adds a contact named `John Doe` to the address book.
```

Comment:
To verify, is this still meant to be "address book" or are we changing it to TutorRec here?

Reply:
Since we limiting the context to Singapore home tutors I thought it would be more fitting to use some Singaporean names instead.

Reply:
Yeah "address book"  doesn't refer to the name of the application but rather the nature of the application.

Reply:
Alright then, I'll approve this